rskj-core/src/main/java/co/rsk/peg/storage/StorageAccessor.java

```diff
@@ -5,15 +5,15 @@
 
 public interface StorageAccessor {
 
-    <T> T safeGetFromRepository(DataWord keyAddress, RepositoryDeserializer<T> deserializer);
+    <T> T safeGetFromRepository(DataWord key, RepositoryDeserializer<T> deserializer);
 
-    <T> void safeSaveToRepository(DataWord addressKey, T object, RepositorySerializer<T> serializer);
+    <T> void safeSaveToRepository(DataWord key, T value, RepositorySerializer<T> serializer);
 
     interface RepositoryDeserializer<T> {
-        T deserialize(byte[] data) throws IOException;
+        T deserialize(byte[] value) throws IOException;
     }
 
     interface RepositorySerializer<T> {
-        byte[] serialize(T object) throws IOException;
+        byte[] serialize(T value) throws IOException;
```

Comment:
Let's discuss this. Not sure it makes sense for serializer and desearializers to thrown an IOException

Reply:
I agree. It would complicate tests. Since we should have full control over tests, there's no need to make this test utility method declare a checked exception.

Reply:
Done! Exceptions were removed.